src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+const calendar = document.querySelector('.calendar');
```

Comment:
It's better to find elements by id instead of class. It's not only faster but also more reliable because ids are unique and classes are not. So, consider adding an id to your calendar element and use `document.getElementById` here.